### PR TITLE
Scadafence to py3

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4172

## Description
Scad afence to py3.

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by sapirshuker** [comment]
Validation *only* failed on [ERROR]: Packs/SCADAfence_CNM/Integrations/integration-SCADAfence_CNM.yml: [BA115] - The file Packs/SCADAfence_CNM/Integrations/integration-SCADAfence_CNM.yml cannot be deleted. Please restore the file.

**Comment by Shellyber** [comment]
Validation error issue - https://jira-hq.paloaltonetworks.local/browse/CIAC-4590

File: Packs/SCADAfence_CNM/ReleaseNotes/1_0_8.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### SCADAfence CNM
+- Updated the Docker image to: demisto/python3:3.10.9.40422.
```
Comment (Shellyber) [inline Packs/SCADAfence_CNM/ReleaseNotes/1_0_8.md:4]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.9.40422*.
```

File: Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+        if res.text:
+            res_msg = res.text
+        res.raise_for_status()
+
+        if not res.text:
+            return None
+        return json.loads(res.text)
+
+    except Exception as e:
+        demisto.debug(f"{e}\n{res_msg}")
```
Comment (Shellyber) [inline Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py:106]:
```suggestion
```

File: Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+def get_alert_severity():
+    """
+    validate severity values provided as parameter
+    :return: set: valid severity values
+    """
+    s = ALERT_SEVERITY.replace(" ", "")
+    s_arr = s.split(",")
+    if sum([x in ['Information', 'Warning', 'Threat', 'Severe', 'Critical'] for x in s_arr]) == len(s_arr):
+        return set(s_arr)
+    demisto.debug('Invalid alert severity values')
```
Comment (Shellyber) [inline Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py:60]:
```suggestion
```

File: Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+    """
+
+    :param method: string: https method
+    :param url_suffix: string: API route
+    :param params_dict: dict: request parameters
+    :param headers: dict: optional http headers
+    :return: dict: response data
+    """
+    req_params = {
+    }
```
Comment (Shellyber) [inline Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py:74]:
```suggestion
    req_params = {}
```

Comment (Shellyber) [inline Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py:74]:
```suggestion
    req_params = {}
```

File: Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+
+        else:
+            if demisto.args()['hostName'] == ep['src_hostname']:
+                ret.append(dest_endpoint(ep))
+            else:
+                ret.append(src_endpoint(ep))
+    return ret
+
+
+# The command demisto.command() holds the command sent from the user.
```
Comment (Shellyber) [inline Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py:281]:
```suggestion
```

Comment (Shellyber) [inline Packs/SCADAfence_CNM/Integrations/SCADAfenceCNM/SCADAfenceCNM.py:281]:
```suggestion
```

**Review by Shellyber — COMMENTED** [review]
Nice

**Review by Shellyber — APPROVED** [review]
Nice

**Pull request merged**